website/cue/reference/releases/0.45.0.cue

```diff
@@ -0,0 +1,332 @@
+				The `chronicle_unstructured` sink now sets the `content-encoding` header when compression is enabled.
+				"""
+			contributors: ["chocpanda"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `tag_cardinality_limit` transform now supports customizing limits for specific metrics, matched by metric name and optionally its namespace.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add `websocket_server` sink that acts as a websocket server and broadcasts events to all clients.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "feat"
+			description: """
+				Sources running HTTP servers (`http_server` source, `prometheus` source, `datadog_agent`, etc.) now support a new `custom` authorization strategy . If a strategy is not explicitly defined, it defaults to `basic`, which is the current behavior.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				The systemd service now validates the config with parameter `--no-environment` on service reload.
+				"""
+			contributors: ["rsrdesarrollo"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `dnstap` source now uses [v20250201](https://github.com/dnstap/dnstap.pb/releases/tag/v20250201) dnstap protobuf schema.
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				Allow users to specify `session_name` when using aws auth.
+				"""
+			contributors: ["akutta"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add support for more chronicle regional endpoints as listed - https://cloud.google.com/chronicle/docs/reference/ingestion-api#regional_endpoints
+				"""
+			contributors: ["chocpanda"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add an option to Google Chronicle sink to set a fallback index if the provided template in the `log_type` field cannot be resolved
+				"""
+			contributors: ["ArunPiduguDD"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add virtual memory metric to the process host metrics collector.
+				"""
+			contributors: ["nionata"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				In datadog search syntax, allow the following queries to match on several fields (OR condition):
+
+				- `tags` will lookup the fields `tags` and `ddtags`
+				- `source` will lookup the fields `source` and `ddsource`
+				"""
+			contributors: ["20agbekodo"]
+		},
+		{
+			type: "fix"
+			description: """
+				`enrichment_table`s loaded from a CSV file with `include_headers: false` no longer drop the first row of data
+				"""
+			contributors: ["B-Schmidt"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The `generate-schema` subcommand accepts an optional `output_path` option.
+				"""
+			contributors: ["pront"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Allow additional types to be used in `tests.inputs.log_fields` values, including
+				nested objects and arrays.
```

Comment:
Glad to see this fixed! That has been a common footgun for people writing tests.